tests/unit/fields/string.spec.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from "@vue/test-utils";
+import { getComponentPrototype } from "@/components/factory";
+
+const localVue = global.localVue;
+const store = global.store;
+
+const wrapper = mount(getComponentPrototype("furet-ui-form-field-string"), {
+  store,
+  localVue,
+  propsData: {
+    resource: {},
+    data: {},
+    config: {}
+  },
+  mocks: {
```

Comment:
[Vue warn]: Injection "partIsReadonly" not found
you must replace **mocks**  by **provide**, need by the inject entry

Reply:
Thanks for this advice ! :)

Reply:
done!